cypress/support/giftCards.po.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference types="Cypress" />
+import { BaseComponent } from './base.component';
+
+export class GiftCardsPo extends BaseComponent {
+    pageUrl = '/gift-cards';
+    giftCardPrintAtHome = `img[alt='Print-at-Home']`;
+    giftPrice100 = '#a-autoid-25';
```

Comment:
autoid - looks like this id generated automatically, so need to find some better one, like: 
cy.get('#gc-amount-mini-picker .a-list-item').contains(expectedPrice).as('price');